src/components/ExerciseForm.jsx

```diff
@@ -51,8 +49,22 @@ export default function ExerciseForm() {
 			<StyledForm onSubmit={handleSubmit} variant="exercise">
 				<Timer minutes={minutes} seconds={seconds} />
 				<div>
-					<InputRange id="min" value={minutes} onChange={handleMinutes} name="minutes" />
-					<InputRange id="sec" value={seconds} onChange={handleSeconds} name="seconds" />
+					<label htmlFor="min">
```

Comment:
The label should contain the information what kind of information the user can add. In This case I would add min or minutes in the label.